content/blog/2018-03-15-update-on-async-rendering.md

```diff
@@ -35,9 +35,9 @@ Before we begin, here's a quick overview of the lifecycle changes planned for ve
 
 `embed:update-on-async-rendering/new-lifecycles-overview.js`
 
-The new static `getSnapshotBeforeUpdate` lifecycle is invoked after a component is instantiated as well as when it receives new props. It should return an object to update `state`, or `null` to indicate that the new `props` do not require any `state` updates.
+The new static `getSnapshotBeforeUpdate` lifecycle is invoked after a component is instantiated as well as when it receives new props. It can return an object to update `state`, or `null` to indicate that the new `props` do not require any `state` updates.
```

Comment:
Real stupid question: shouldn't this line be `getDerivedStateFromProps` ?

Reply:
Yes, thanks!